<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>713</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>713</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-dictionary"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="WordsListWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="layoutMode">
     <enum>QListView::LayoutMode::SinglePass</enum>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="WordLabel">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: 10px;
padding-top: 35px;
padding-left: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Manzana</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TopicLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>еда</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing</set>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TranslationLabel">
       <property name="font">
        <font>
         <family>Sitka</family>
         <pointsize>22</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 10px; </string>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Яблоко</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="ExampleLabel">
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>A mi me gusta comer las manzanas cada dia.</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ExampleTranslationLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Мне нравится есть яблоки каждый день.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="SearchLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Поиск по слову или теме...</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseWordLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Информация о выбранном слове будет отображаться здесь.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>WordsListWidget</zorder>
   <zorder>SearchLineEdit</zorder>
   <zorder>chooseWordLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="menuOptionLoadData"/>
    <addaction name="menuOptionSaveData"/>
    <addaction name="menuOptionSortData"/>
    <addaction name="menuOptionTopicSortData"/>
   </widget>
   <widget class="QMenu" name="menuWord">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Слово</string>
    </property>
    <addaction name="menuOptionAddWord"/>
    <addaction name="menuOptionDeleteWord"/>
    <addaction name="menuOptionEditWord"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuWord"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuOptionAddWord">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="menuOptionDeleteWord">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="menuOptionEditWord">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="menuOptionLoadData">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="menuOptionSaveData">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="menuOptionSortData">
   <property name="text">
    <string>Сортировать по алфавиту</string>
   </property>
  </action>
  <action name="menuOptionTopicSortData">
   <property name="text">
    <string>Сортировать по теме</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
